src/main/webapp/app/shared/layouts/profiles/profile.service.ts

```diff
@@ -73,11 +77,19 @@ export class ProfileService {
     }
 
     private mapAllowedOrionVersions(data: any, profileInfo: ProfileInfo) {
-        profileInfo.allowedMinimumOrionVersion = data['allowed-minimum-orion-version'];
+        if (data['allowed-minimum-orion-version']) {
+            profileInfo.allowedMinimumOrionVersion = data['allowed-minimum-orion-version'];
+        } else {
+            profileInfo.allowedMinimumOrionVersion = data['allowedMinimumOrionVersion'];
```

Comment:
I don't really understand the else clause here? Why should the key change to camel case?

Reply:
I use camel case for this property because there is no such property with the name `allowed-minimum-orion-version` in `profile-info.model.ts`. But there is one in camel case. Therefore the non camel case one will not be found and return undefined. This could lead to an error further on. 

The reason why I want to start and continue supporting the non camel case one (in the 'if-then' statement) is that it seems to be working so far but it does not come from `profile-info.model.ts` natively but was most likely appended somewhere else. If I remove this option other parts of the code might break.
![image](https://user-images.githubusercontent.com/65814168/119275827-882f4100-bc17-11eb-9746-71ea1ad014a5.png)
I hope my reasoning become more clear?
(If I am wrong please correct me.)

Reply:
data is coming from the server. It won't have a camelCase style property. That's why we need to convert after all

Reply:
Yes, you are absolutely right, after using Postman to receive an actual server response I understood the issue and the reason for `profile.service`.  Thus, I removed the `if-else` blocks.